homebrew_challenge/src/components/Footer/Footer.jsx

```diff
@@ -0,0 +1,135 @@
+import React from "react";
+import "./Footer.css";
+import Container from "react-bootstrap/Container";
+import Row from "react-bootstrap/Row";
+import Col from "react-bootstrap/Col";
+
+const Footer = () => {
+  return (
+    <footer>
+      <Container fluid className="font-small">
+        <Row className="p-3 d-flex justify-content-center">
+          <Col xs={12} md={3}>
+            <div className="sitemap">
+              <div className="title">CONNECT WITH US</div>
+              <div className="entry link">
+                <a className="entry link" href="https://www.linkedin.com/company/scottish-tech-army-limited">
+                  LinkedIn
+                </a>
+              </div>
+              <div className="entry link">
+                <a
+                  className="entry link"
+                  href="https://twitter.com/ScotTechArmy"
```

Comment:
Shouldn't these links be target="_blank".... ?

Reply:
all updated